vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -236,6 +236,10 @@ function Install-ContainerD {
     # and the containerd to managed customer containers after provisioning the vm is not necessary
     # the one used here, considering containerd version/package is configurable, and the first one
     # is expected to override the later one
+    $buildNumber = (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").CurrentBuild
```

Comment:
Would it not work if v1.6.21 is installed inside 23h2 vhd and v1.7.9 is installed in provisioning 23h2 nodes?